data/actionsmap/yunohost.yml

```diff
@@ -226,6 +226,97 @@ user:
                 username:
                     help: Username or email to get information
 
+#############################
+#           Alias            #
+#############################
+alias:
+    category_help: Manage alias
+    actions:
+
+        ### alias_list()
+        list:
+            action_help: List existing aliases
+            api: GET /alias
+            configuration:
+                authenticate: all
+                authenticator: ldap-anonymous
```

Comment:
removing "authenticator: ldap-anonymous" is fine for me, but it can be painful for a cli administrator to enter password each time just to list aliases